### DISCUSSION TITLE
posts/ruankao-architect/

### DISCUSSION THREAD

**Discussion by giscus** [jerrylususu/jerrylususu.github.io#31] [Blog Comments]
# posts/ruankao-architect/

运气好的话，60 个小时就够了

https://nekonull.me/posts/ruankao-architect/

**Comment by JuncusEffusus** [comment]
恰好看到了，就补充一下~应该不是退税3600元，而是3600的免税额度，所以金额大概是3600*本年度最高税率（uu看到后麻烦删除我这条评论

**Reply by jerrylususu** [reply]
感谢指正，不过 Github 的设计我没法主动删除评论，你可以自己删除 😂

**Comment by SmallerPig** [comment]
> 其中基础知识和案例分析上午连着考，论文则是下午单独考.

作者你确定你考的是 24 年上半年的考试么？

**Reply by jerrylususu** [reply]
是的，可以参考我的准考证截图上的时间。可能是不同考区的时间安排不同（我在广东深圳）？欢迎分享你的体验。
![IMG_20240809_091403](https://github.com/user-attachments/assets/26eb1c33-cd75-4c7d-9dce-0694ce23f3e5)

**Reply by jeffreykbrooks** [reply]
从24年开始改版，一年两次了

**Comment by mingzaily** [comment]
阮老师那过来的，佩服 OP，我自从大学考过中级软件设计师以后，一直想找机会考，但一直没下定决心，下午的论文科目有点劝退我

**Reply by simpleyzh** [reply]
作为开发，我上半年报名了高项，论文让我望而却步了。

**Comment by BaoXuebin** [comment]
很好的备考分享，感谢

**Comment by chaggle** [comment]
刚好下半年准备考个软考架构师，感谢博主分享

**Reply by zhihui-Yu** [reply]
我也在想考，一方面为了整理知识，一方面有点担心论文。有点强迫症

**Comment by shawn-bluce** [comment]
这次也上了阮老师的[周刊](https://www.ruanyifeng.com/blog/2024/08/weekly-issue-312.html) 🤣

**Reply by jerrylususu** [reply]
是自己投稿的 🤣

**Comment by guowei-gong** [comment]
這篇文章對我有幫助，謝謝

**Comment by imbant** [comment]
个人推荐下B站 zst_2001，也讲得不错

**Comment by Ai01** [comment]
这个证在杭州可以买人才房，节约200w。可以关注一下。

**Comment by YCF** [comment]
论文是不是手写，我字写得不好是不是没戏。

**Reply by jerrylususu** [reply]
已经全面电子化了，打字就行

**Comment by Easterlings** [comment]
非常相似的经历，上半年也在深圳考了高级架构师，但是由于一直没有进入备考状态，准备不足，最后只做了前两科的不充分的备考，论文完全没写过，上午的考试结束后算了下案例分析必过不了，直接放弃了。之后有机会再考吧。

**Comment by hsxrcn** [comment]
我想咨询下，阿卜杜拉国王科技大学这个研究生读研，需要花费大概多少￥？对工作是否有帮助？

**Reply by jerrylususu** [reply]
我是全奖，所以没花钱，算上生活费补贴还赚了点 😂 ；我秋招（21年）的时候可能有点用，不过工作上其实用不太到特定的知识点，更多是“知道有这么个坑”的感觉

**Comment by RogersDong** [comment]
打字的时候，是否支持五笔输入法？

**Reply by jerrylususu** [reply]
输入准考证和个人信息登录之后之后有个设备测试页，可以选系统里输入法；我这边的考场有搜狗拼音和搜狗五笔可选。

**Comment by captainIT** [comment]
这有部分真题
https://ebook.qicoder.com/%E7%B3%BB%E7%BB%9F%E6%9E%B6%E6%9E%84%E8%AE%BE%E8%AE%A1%E5%B8%88/

**Comment by limng06** [comment]
报了名最后没去，33岁满脑子就是躺平。。哎，快救救我

**Comment by fengzhao** [comment]
你刷题一般是用什么软件呢？

**Reply by jerrylususu** [reply]
基本就是 软考达人 小程序
